### DISCUSSION TITLE
Is there any API available using which I can identify the split brain situation.

### DISCUSSION THREAD

**Discussion by dipaktank** [rabbitmq/rabbitmq-server#5787] [Questions] [answered]
Hi

Is there any API available using which I can identify the split brain situation. So when network partition happened, if I hit that API than it should provide information that nodes are working individually.

Thanks

**Comment by michaelklishin** [comment, marked as answer]
Clusters of two nodes are [explicitly not supported](https://www.rabbitmq.com/clustering.html#node-count).

There are node metrics, including partitions in effect, exposed by the HTTP API.
With Prometheus all metrics are node-local, so the list of nodes becomes the list of nodes that report metrics. IIRC partitions in effect is still one of the exposed metrics.

The definition of "down" in distributed systems is broad, so other metrics can be of high relevance as well.

**Comment by michaelklishin** [comment]
`GET /api/nodes` is the HTTP API endpoint that would be a good start. With Prometheus, all the metrics relevant to this question are provided under `GET /metrics`.
